scheduler/scheduler-api/src/test/java/org/ow2/proactive/scheduler/common/util/PaginationTest.java

```diff
@@ -0,0 +1,59 @@
+package org.ow2.proactive.scheduler.common.util;
+
+import org.junit.Test;
+
+import static com.google.common.truth.Truth.assertThat;
```

Comment:
@marcocast Truth is at least as readable as Hamcrest, sometimes more, but in all cases more simple to use. Since new tools and library have been introducing by some people in their developments without any consensus, I do the same when I think it can ease my work.